pulp_rpm/app/models.py

```diff
@@ -110,7 +110,7 @@ class Package(Content):
 
     # Required metadata
     name = models.TextField()
-    epoch = models.TextField()
+    epoch = models.TextField(blank=True)
```

Comment:
Why does this need blank=True but nowhere else? Does it raise an exception without this?

Reply:
The other fields (name, version, release, arch) are required 100% of the time, but epoch can be assumed to be null unless otherwise specified.

Reply:
That makes sense, but how does that work with our default assignment here:  https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/models.py#L219 Maybe epoch should default to '0' instead of ''? Or maybe that other line is wrong?

Reply:
Epoch can either be excluded entirely (which behaves as though it were zero) or it can be explicitly set (and become part of the filename).  So we need to handle both cases instead of default to zero.

Reply:
https://fedoraproject.org/wiki/Packaging:Guidelines?rd=PackagingGuidelines#Use_of_Epochs

Reply:
I think the `or '0'` in the other PR is wrong and this PR is right. We want an unspecified value to receive a default of '' and not None. By using blank=True without `null=True` we are avoiding confusion between None versus '' for defaults.

Can this PR fix that along w/ any smash related fixes?